pygotham/frontend/templates/talks/proposal.html

```diff
@@ -13,6 +13,9 @@
 
         <div class="row">
           {{ wtf.horizontal_field(form.name, placeholder='Title') }}
+          <small>
+            Got a Co-Presenter? <a href="mailto:organizers@pygotham.org" >Let us know!</a>
```

Comment:
I'm still not a fan of this approach. This really should be done with a way to select a user.